### PR TITLE
Todo list: ask review

### PR DESCRIPTION
**Pull request by LionRouge1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+> This project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Demo 
+Click here to see [Live](https://lionrouge1.github.io/To_do_list/)
+
+## Built With
```
Comment (misheck12) [inline README.md:10]: - [x] please add webpack to this list as it helps understand the tools uses.

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
-# Todo_review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+> This project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Demo 
+Click here to see [Live](https://lionrouge1.github.io/To_do_list/)
```
Comment (misheck12) [inline README.md:8]: Removing this live link can help but you can also put the live link

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+- Press Enter to create your local clone.
+- run ``cd To_do_list``
+- now you an open the project with any code editor and go live or simply open the `dist/index.html` page with the browser.
+
+### Prerequisites
+
+- GitHub
+- Git
+
+## Authors
```
Comment (misheck12) [inline README.md]: - [x] please change this from authors to author

File: drag&drop.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
+import TDlist from './TDlist.js';
```
Comment (misheck12) [inline drag&drop.js:1]: - [x] Next time when naming please you small letters.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <ul class="items" id="container_items">
+    <!-- list items -->
+    </ul>
+    <div class="contain_footer">
+     <p id="clear">Clear all completed</p>
+    </div>
+   </section>
+  </main>
+  <footer>
+    <p>@ 2022 made by <a href="https://github.com/LionRouge1">@LionRouge1</a></p>
```
Comment (misheck12) [inline index.html]: - [x] you can use &copy; instead of @ to make sure your work is perfect

**Review by misheck12 — CHANGES_REQUESTED** [review]
Hey @LionRouge1 

Good work so far! It appears that you are close to completing the project, but there are a few issues that still need to be addressed before moving on to the next one.

First, let me commend you on:

- A great commit history. ✔️
- Passing all Linters. ✔️
- A professional pull request. ✔️
Keep up the good work, you're doing great! 👏

Please refer to the changes that I have listed and work on them before proceeding to the next project. Keep up the good work, you're almost there!




 ## Required Changes  :recycle:

File: README.md (hunk 1 above)
Comment (LionRouge1) [reply README.md:10]: Ok thanks I will

File: README.md (hunk 2 above)
Comment (LionRouge1) [reply README.md:8]: 👍

File: README.md (hunk 3 above)
Comment (LionRouge1) [reply README.md]: 👍

File: drag&drop.js (hunk 4 above)
Comment (LionRouge1) [reply drag&drop.js:1]: Ok I understand what you mean. But the 'TD' refers to the project name.

File: README.md (hunk 2 above)
Comment (misheck12) [reply README.md:8]: Do you mind adding the link to your live page here

File: README.md [hunk 6]
```diff
@@ -1 +1,62 @@
...
+# To Do list
+
+> This project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Demo 
+Click here to see [Live](https://lionrouge1.github.io/To_do_list/)
+
+## Built With
+
+- HTML
```
Comment (misheck12) [inline README.md:12]: Add webpack after javascript here like this 

- html
- css
- javascript
- webpack

File: README.md (hunk 3 above)
Comment (misheck12) [reply README.md]: I have noticed you forgot to change this one from Authors to Author

**Review by misheck12 — APPROVED** [review]
Hi @LionRouge1 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! tada

**### To highlight**

1. No linter errors :heavy_check_mark:

2. Good code design and implementation.:heavy_check_mark:

3. Add all the requirements.:heavy_check_mark:

image


Cheers and Happy coding! clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**